app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,80 @@
+class TweetsController < ApplicationController
+
+  # Index action
+  get '/tweets' do 
+    if logged_in?
+      @tweets = Tweet.all
+      erb :"tweets/tweets"
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :"tweets/create_tweet"
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets' do 
+    if params[:content] == ""
+      redirect "/tweets/new"
+    else
+      user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(:content => params[:content], :user_id => user.id)
```

Comment:
The best way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.